### ISSUE TITLE
SwiftGen 6.2.0 generates default.profraw files

### ISSUE THREAD

**Issue by marcelofabri** [SwiftGen/SwiftGen#719]
I've just updated to 6.2.0 and now I'm seeing `default.profraw` files being generated whenever I run `swiftgen`. 

`default.profraw` is apparently related to code coverage and generated by LLVM.

Maybe I introduced it in https://github.com/SwiftGen/SwiftGen/commit/1695642e24a3844434020a4acc230efd9f95cb43?

**Comment by djbe** [comment]
How did you install SwiftGen? Tried it really quickly via Mint here, and there's no `profraw` file that I can see

**Comment by marcelofabri** [comment]
I've installed from the binary in GitHub releases

**Comment by djbe** [comment]
Well, can confirm it also happens when installing via CocoaPods, so any build generated via `xcodebuild`.

**Comment by JonathanDowning** [comment]
I've added a workaround for this in our Xcode project build run script phase in the meantime!
``` bash
if which swiftgen >/dev/null; then
swiftgen
rm -f "$SRCROOT/default.profraw" # Workaround for Swiftgen 6.2.0 producing this file.
else
echo "warning: SwiftGen not installed, download it from https://github.com/SwiftGen/SwiftGen"
fi
```

**Comment by djbe** [comment]
Should be fixed in SwiftGen 6.2.1 🎉
